### PR TITLE
R.swift

### PR DESCRIPTION
**Pull request by osatishur**

### CODE REVIEW

File: ADMVEDU95/ADMVEDU95/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -9,11 +9,11 @@
 "Start searching" = "Start searching";
 "song" = "song";
 "feature-movie" = "feature-movie";
-"Song" = "Song: %@";
+"_Song" = "Song: %@";
```
Comment (obazyleva) [inline ADMVEDU95/ADMVEDU95/en.lproj/Localizable.strings]: why is underscore needed?

File: ADMVEDU95/R.generated.swift [hunk 2]
```diff
@@ -0,0 +1,1554 @@
+//
```
Comment (obazyleva) [inline ADMVEDU95/R.generated.swift]: this file should be git-ignored

File: ADMVEDU95/ADMVEDU95/en.lproj/Localizable.strings (hunk 1 above)
Comment (osatishur) [reply ADMVEDU95/ADMVEDU95/en.lproj/Localizable.strings]: Because for R.swift "song" and "Song" is equal

Comment (obazyleva) [reply ADMVEDU95/ADMVEDU95/en.lproj/Localizable.strings]: in general keys should be structured kind of "X.Y.Z", e.g. "Search.Song.Title" and "Filters.Song.Title" or "Details.Song.Title" etc
Please structure all localisable keys

**Comment by obazyleva** [comment]
Additionally please update swift format script - it should work as linter

File: ADMVEDU95/.swiftlint.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+disabled_rules:
+  - trailing_whitespace
```
Comment (obazyleva) [inline ADMVEDU95/.swiftlint.yml]: no disabled rules in SwiftLint config

File: ADMVEDU95/ADMVEDU95/Extensions/ViewController+ShowAlert.swift [hunk 4]
```diff
@@ -9,18 +9,26 @@ import UIKit
...
 
-        alert.addAction(UIAlertAction(title: "Cancel", style: .cancel))
-        self.present(alert, animated: true, completion: nil)
+    func showAlertWithRetry(message: String, completion: @escaping () -> Void) {
+        let alert = UIAlertController(title: R.string.localizable.alertErrorTitle(),
+                                      message: message,
+                                      preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: R.string.localizable.alertRetryAction(),
+                                      style: .default,
+                                      handler: { (_: UIAlertAction!) in
```
Comment (obazyleva) [inline ADMVEDU95/ADMVEDU95/Extensions/ViewController+ShowAlert.swift]: handler: { _ in 
    completion()
}

File: ADMVEDU95/ADMVEDU95/en.lproj/Localizable.strings [hunk 5]
```diff
@@ -6,69 +6,75 @@
...
+//Category
 "song" = "song";
-"feature-movie" = "feature-movie";
-"Song" = "Song: %@";
-"Artist" = "Artist: %@";
-"Album" = "Album: %@";
-"Director" = "Director: %@";
-"Movie" = "Movie: %@";
-"no info" = "no info";
 "all" = "all";
```
Comment (obazyleva) [inline ADMVEDU95/ADMVEDU95/en.lproj/Localizable.strings]: why are these keys not complex?

File: ADMVEDU95/ADMVEDU95.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -627,7 +627,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\n\"${PODS_ROOT}/SwiftFormat/CommandLineTool/swiftformat\" --lint \"$SRCROOT\"\n";
+			shellScript = "# Type a script or drag a script file from your workspace to insert its path.\n\"${PODS_ROOT}/SwiftFormat/CommandLineTool/swiftformat\" \"$SRCROOT\" \n";
```
Comment (obazyleva) [inline ADMVEDU95/ADMVEDU95.xcodeproj/project.pbxproj]: where is the "lint" option?

File: ADMVEDU95/ADMVEDU95/Models/Category.swift [hunk 7]
```diff
@@ -15,5 +15,31 @@ enum Category: String, CaseIterable {
     case tvShow
     case ebook
     case audiobook
-    case musicVideo = "musicVideoPlural"
+    case musicVideo
+    case noInfo
+}
+
+extension Category {
+    var kind: String {
```
Comment (obazyleva) [inline ADMVEDU95/ADMVEDU95/Models/Category.swift]: would be better description or title

**Pull request closed without merging**